params/config.go

```diff
@@ -217,21 +218,45 @@ var (
 		BloomRoot:    common.HexToHash("0xff45a6f807138a2cde0cea0c209d9ce5ad8e43ccaae5a7c41af801bb72a1ef96"),
 	}
 
+	// AstorChainConfig is the chain parameters to run a node on the Ethereum Classic main network.
+	AstorChainConfig = &ChainConfig{
+		ChainID:             big.NewInt(5),
```

Comment:
Same as network ID, please don't use 5, use something two-digit to avoid collisions with existing networks